app/src/components/FormControl/FormControl.tsx

```diff
@@ -1,21 +1,39 @@
 import React from 'react';
 import {Formik, Form, Field, ErrorMessage} from 'formik';
-import {FormControlProps, FormControlFields, FormControlInputFields} from './types';
+import {FormControlProps, FormControlFields, FormControlInputFields, FormFieldComponentProps} from './types';
```

Comment:
тут можно добавить import **type** ...
```suggestion
import type {FormControlProps, FormControlFields, FormControlInputFields, FormFieldComponentProps} from './types';
```